### PR TITLE
New doc- CrowdStrike Falcon FileVantage (Apps)

### PR DESCRIPTION
**Pull request by amee-sumo**
## Purpose of this pull request

This pull request is to create a new apps doc - CrowdStrike Falcon FileVantage

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-586

### CODE REVIEW

File: docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+The Sumo Logic app for CrowdStrike Falcon FileVantage is an essential tool for security teams. It provides the visibility and intelligence needed to detect and respond to file-related threats, ensuring data integrity and compliance across the organization.
+
+:::info
+This app includes [built-in monitors](#crowdstrike-falcon-filevantage-monitors). For details on creating custom monitors, refer to [Create monitors for CrowdStrike Falcon FileVantage app](#create-monitors-for-crowdstrike-falcon-filevantage-app).
+:::
+
+## Log types
+
+This App uses Sumo Logic’s [CrowdStrike FileVantage Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-filevantage-source/) to collect the FileVantage logs from CrowdStrike.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md:35]:
```suggestion
This App uses Sumo Logic’s [CrowdStrike FileVantage Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-filevantage-source/) to collect the FileVantage logs from CrowdStrike.
```

File: docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+Key metrics include file change types (**WRITE**, **RENAME**, **READ** **PERMISSION**, **OTHERS**, **DELETE**), displayed in an easy-to-read pie chart for quick analysis. The dashboard tracks changes by operating system, identifies top hosts and users making changes, and highlights threats or anomalies.
+
+The dashboard also shows the monitoring rules that triggered the most changes, assisting in policy refinement. A detailed table of recent file activities, including timestamps, hostnames, file paths, severities, and associated policies, is provided. Overall, the dashboard strengthens the detection and response to unauthorized or suspicious file activities. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Crowdstrike+Falcon+FileVantage/Crowdstrike+Falcon+FileVantage+dashboard.png')} alt="CrowdStrike Falcon FileVantage Overview" />
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md:138]:
```suggestion
The dashboard also shows the monitoring rules that triggered the most changes, assisting in policy refinement. A detailed table of recent file activities, including timestamps, hostnames, file paths, severities, and associated policies, is provided. Overall, the dashboard strengthens the detection and response to unauthorized or suspicious file activities. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Crowdstrike+Falcon+FileVantage/Crowdstrike+Falcon+FileVantage+dashboard.png')} alt="CrowdStrike Falcon FileVantage Overview" style={{border: '1px solid gray'}} width="800" />
```

File: docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+Additionally, security analysts can review file deletions with specific details, including file paths, user names, and originating countries. The severity trends are visually represented, providing a clear overview of security incidents over time and helping teams identify escalation points.
+
+While the dashboard provides placeholders for tracking malicious file changes and directory changes, data in these sections requires specific configuration based on the environment. Finally, the **Host-Based Change Monitoring** table offers detailed insights into the hosts and host groups involved in file changes, assisting in pinpointing areas that need further investigation. Overall, this dashboard is a vital tool for ensuring file integrity and effectively mitigating risks. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Crowdstrike+Falcon+FileVantage/Crowdstrike+Falcon+FileVantage+Security.png')} alt="CrowdStrike Falcon FileVantage Security" />
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/crowdstrike-falcon-filevantage.md:149]:
```suggestion
While the dashboard provides placeholders for tracking malicious file changes and directory changes, data in these sections requires specific configuration based on the environment. Finally, the **Host-Based Change Monitoring** table offers detailed insights into the hosts and host groups involved in file changes, assisting in pinpointing areas that need further investigation. Overall, this dashboard is a vital tool for ensuring file integrity and effectively mitigating risks. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Crowdstrike+Falcon+FileVantage/Crowdstrike+Falcon+FileVantage+Security.png')} alt="CrowdStrike Falcon FileVantage Security" style={{border: '1px solid gray'}} width="800" />
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
LGTM, except the images uploaded to the S3 bucket. The dashboard images do not have the required clarity while viewing the file. Can you please export the dashboard image from the platform and upload it to S3?
Screenshot 2024-12-09 at 2 20 36 PM

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by jpipkin1** [comment]
Publishing this PR at the request of @ankitgoelcmu.

**Pull request merged**